{"uuid": "e1e92fc0-8b7b-4ab6-a0ad-306e84f0c1ca", "children": ["413daad8-7b8c-4461-829d-b061d8d7fc3a", "4c62e22f-111f-4550-9592-6886d9e14360", "f5c9c857-1305-4755-a5d6-b66c1a949b00", "4758280d-081f-4628-baaf-3b2b34d88e4a", "efa2df73-ce41-4f68-b4b1-bae2404976fb", "0237f5d1-164c-4c8d-a17c-ed61c0869990", "b0422fe9-abf2-4d38-9b23-13d0a6503c0a"], "befores": [{"name": "_Class__pytest_setup_class", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"E:\\python37\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"E:\\python37\\lib\\site-packages\\_pytest\\fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"E:\\python37\\lib\\site-packages\\_pytest\\fixtures.py\", line 788, in call_fixture_func\n    res = next(it)\n  File \"E:\\python37\\lib\\site-packages\\_pytest\\python.py\", line 701, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"E:\\python37\\lib\\site-packages\\_pytest\\python.py\", line 632, in _call_with_optional_argument\n    func(arg)\n  File \"E:\\pytest_study\\bdrack1.4.6\\testsuit\\cstor_Exam_process.py\", line 29, in setup_class\n    cls.dl.admin_login('admin', 'cstorfs')\n  File \"E:\\pytest_study\\bdrack1.4.6\\base\\base_login.py\", line 34, in admin_login\n    self.input_text(self.username, username)\n  File \"E:\\pytest_study\\bdrack1.4.6\\base\\base_config.py\", line 55, in input_text\n    self.base_element(locator).clear()\n  File \"E:\\pytest_study\\bdrack1.4.6\\base\\base_config.py\", line 44, in base_element\n    return WebDriverWait(self.driver, timeout, poll_frequency).until(lambda x: x.find_element(*locator))\n  File \"E:\\python37\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "打开浏览器", "status": "passed", "start": 1654586129796, "stop": 1654586130158}, {"name": "输入用户名", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"E:\\pytest_study\\bdrack1.4.6\\base\\base_login.py\", line 34, in admin_login\n    self.input_text(self.username, username)\n  File \"E:\\pytest_study\\bdrack1.4.6\\base\\base_config.py\", line 55, in input_text\n    self.base_element(locator).clear()\n  File \"E:\\pytest_study\\bdrack1.4.6\\base\\base_config.py\", line 44, in base_element\n    return WebDriverWait(self.driver, timeout, poll_frequency).until(lambda x: x.find_element(*locator))\n  File \"E:\\python37\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1654586130158, "stop": 1654586136832}], "start": 1654586128762, "stop": 1654586136846}], "start": 1654586128762, "stop": 1654586137033}